src/pages/documentation/release-notes/index.md

```diff
@@ -4,6 +4,24 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## June 20, 2023
+
+## AEP React Native libraries 5.0.0
+
+The following React Native libraries have been upgraded to use the Adobe iOS SDK 4.x. Starting with version 5.0.0 of the Adobe React Native SDK, all React Native libraries that share the same major version are compatible with each other.
+
+ - [@adobe/react-native-aepcore => 5.0.0](https://www.npmjs.com/package/@adobe/react-native-aepcore/v/5.0.0)
```

Comment:
for consistency with the previous notes, let's use the `libname:5.0.0` notation, e.g: `@adobe/react-native-aepcore:5.0.0`